CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ if(NOT CMAKE_CONFIGURATION_TYPES)
 endif()
 
 option(QT6 "Build with Qt6" ON)
-option(QML "Build with QML" OFF)
+option(QML "Build with QML" ON)
```

Comment:
Use a CMake dependent option that only sets this to on of Qt6 is also enabled

Reply:
Am I right in thinking than the condition check after isn't required anymore as `QML` will be forced to `OFF` if `QT6=OFF`?

Reply:
Yes.